src/App.js

```diff
@@ -1,5 +1,72 @@
+import { Input } from './View/inputViews.js';
+import { Output } from './View/outputViews.js';
+
+import { countPurchaseAmount } from './Model/purchaseCountCalculator.js';
+import { generatedLottoList } from './Model/lottoListGenerator.js';
+import { WinningStatistics } from './Model/winningStatisticsCalculator.js';
+import { calculateRateOfReturn } from './Model/rateOfReturnCalculator.js';
+
 class App {
-  async run() {}
+  constructor() {
+    this.input = new Input();
+    this.output = new Output();
+
+    this.purchaseAmount = null;
+    this.purchaseCount = null;
+    this.lottoList = null;
+    this.winningNumbers = null;
+    this.paredWinningNumbers = null;
+    this.bonusNumber = null;
+    this.rateOfReturn = null;
+  }
+
+  async run() {
+    await this.getPurchaseAmount();
+    this.printPurchaseCount();
+    this.printLottoList();
+
+    await this.getWinningNumbers();
+    await this.getBonusNumber();
+
+    this.printWinningAmount();
+    this.printRateOfReturn();
+  }
```

Comment:
run 내부 코드의 가독성에 대해서 고민을 하셨다 하셨는데, 개인적으로는 함수별로 기능이 명확하고 중간중간 개행을 넣어 주셔서 읽기 어렵지 않았습니다.
고민하셨던 이유가 아마도 함수들을 호출하는 코드들이 늘어져 있었기 때문이 아닐까 싶긴 한데,,, 저였다면 아래처럼 개행으로 나누신 함수 호출들을 또 다른 메소드들로 묶어서 좀 더 큰 단위의 역할들로 쪼갰을 것 같네요!

```suggestion
  async run() {
    await #purchase();
  }
  
  async #purchase() {
    await this.getPurchaseAmount();
    this.printPurchaseCount();
    this.printLottoList();
  }
```

Reply:
@yoouyeon 
정연님 의견 나눠주셔서 감사드려요 🙏
저도 이 부분을  처음에는 함수들을 한 메소드로 묶어 보기도 하였는데, 그렇게 되니 또 함수 네이밍이 애매하더라구요! 

예를 들어, 
    await this.getPurchaseAmount();
    this.printPurchaseCount();
    this.printLottoList();

와 같은 함수들을 묶으면 결국 3개 함수의 의미를 모두 알려주어야 코드 흐름이 한 번에 보일 것 같은데.. 
협업을 한다는 입장에서 어떤 부분이 더 이해가 쉬울 것 같은지 고민이 되네요! 

정연님께서 협업을 하시는 입장에서 볼때는 어떤 부분이 더 가독성이 좋아보이실 것 같은지 궁금합니다! 🤔

Reply:
음... 저는 함수 이름을 너무 구체적으로 지으려고 하지 않는 편이라서
저였다면 `purchase` 정도로 간단하게 함수 이름을 지었을 것 같습니다!

"구매"라는 큰 기능 아래에 
구매할 금액을 입력받고 / 구매한 개수를 보여주고 / 구매한 목록을 보여주는 이 3가지 기능이 포함될 수 있다고 생각하기 때문이기도 하구요.

함수의 기능이 너무 기능과 밀접하게 연결되어 있다면 (예를 들면 `getLottoAmountAndPrintPurchasedLottoInfo` 이런 식으로 ?)
만약에 출력 기능이 빠진다거나 아니면 뭔가 더 추가로 입력받는 것이 생긴다거나 하는 변경사항이 생기면 이름이 어색해져서 함수 이름까지도 고쳐야 하는 상황이 발생할 것 같기도 해서요!

가독성 측면에서도 어플리케이션의 시작 지점인 run 메소드에서는
구매하고 / 당첨 & 보너스 번호를 입력하고 / 결과를 출력한다
정도의 흐름만 파악되어도 괜찮을 것 같다는 생각이 듭니다!